recipes/rply/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "rply" %}
+{% set version = "0.7.4" %}
+{% set sha256 = "723303d6c5f05a7ee19f59531f66c8c7d41cfaef2676057369db1eb5520b378b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/f8/f4/c1fc57461dda9370e9600933162afb144429f7296fe7aa9bd8c8bc562934/{{ name }}-{{ version }}.tar.gz
```

Comment:
Use the url of the form:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18